commands/cluster-delslotsrange.md

```diff
@@ -1,13 +1,14 @@
-The `CLUSTER DELSLOTSRANGE` command is similar to the `CLUSTER DELSLOTS` command in that they both remove hash slots from the node. The difference is that `DELSLOTS` takes a list of hash slots to remove from the node, while `DELSLOTSRANGE` takes a list of slot ranges (specified by start and end slots) to remove to the node.
+The `CLUSTER DELSLOTSRANGE` command is similar to the `CLUSTER DELSLOTS` command in that they both remove hash slots from the node.
+The difference is that `CLUSTER DELSLOTS` takes a list of hash slots to remove from the node, while `CLUSTER DELSLOTSRANGE` takes a list of slot ranges (specified by start and end slots) to remove from the node.
```

Comment:
to remove to the node.  -> to remove from the node.
Anyway, the whole line has been changed, add a newline...